pkg/controllers/nsxserviceaccount/nsxserviceaccount_controller.go

```diff
@@ -253,8 +253,6 @@ func (r *NSXServiceAccountReconciler) updateNSXServiceAccountStatus(ctx *context
 	obj := o
 	if e != nil && *e != nil {
 		obj = o.DeepCopy()
-		obj.Status.Phase = nsxvmwarecomv1alpha1.NSXServiceAccountPhaseFailed
```

Comment:
why we remove status field from serveracount? is there CRD change? we has better to double confirm with @gran-vmv

Reply:
I think we should not remove this since it will block the consumer logic.

Reply:
Maybe we can add a comment that it is being used and requires change on both side.

Reply:
Done, thanks